src/apis/kord-fi/index.ts

```diff
@@ -0,0 +1,157 @@
+import axios from 'axios';
+import { BigNumber } from 'bignumber.js';
+import { Observable, catchError, from, map, of } from 'rxjs';
+
+import { EarnOpportunityTokenStandardEnum } from 'src/enums/earn-opportunity-token-standard.enum';
+import { EarnOpportunityTypeEnum } from 'src/enums/earn-opportunity-type.enum';
+import { EarnOpportunityToken } from 'src/interfaces/earn-opportunity/earn-opportunity-token.interface';
+import { SavingsItem } from 'src/interfaces/earn-opportunity/savings-item.interface';
+import { UserStakeValueInterface } from 'src/interfaces/user-stake-value.interface';
+import { ExchangeRateRecord } from 'src/store/currency/currency-state';
+import { TEZ_TOKEN_METADATA, TEZ_TOKEN_SLUG, TZBTC_TOKEN_METADATA } from 'src/token/data/tokens-metadata';
+import { getTokenSlug } from 'src/token/utils/token.utils';
+import { tzktUrl } from 'src/utils/linking.util';
+import { tzToMutez } from 'src/utils/tezos.util';
+
+import { fallbackTezosToolkit } from '../youves/utils';
+import { KordFiLendStats, KordFiStatsResponse, KordFiUserDepositsResponse } from './types';
+
+const kordFiApi = axios.create({
+  baseURL: 'https://back-llb-beta.kord.fi'
+});
+
+const TEZOS_CONTRACT_ADDRESS = 'KT19qWdPBRtkWrsQnDvVfsqJgJB19keBhhMX';
+const TZBTC_CONTRACT_ADDRESS = 'KT1WL6sHt8syFT2ts7NCmb5gPcS2tyfRxSyi';
+const TEZOS_TOKEN: EarnOpportunityToken = {
```

Comment:
Sorry, could you create a function like `toEarnOpportunityToken` in `src/utils/earn.utils.ts` and use it for both tokens?